lighthouse-plugin-publisher-ads/messages/en-US.json

```diff
@@ -138,5 +200,10 @@
   "ERRORS": {
     "AREA_LARGER_THAN_VIEWPORT": "Calculated ad area is larger than viewport",
     "VIEWPORT_AREA_ZERO": "Viewport area is zero"
+  },
+  "GROUPS": {
```

Comment:
Should this be `Groups` since it's an enum?

https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming

Reply:
I guess we do this for all top-level keys so it's fine as-is